Header header
string taskType
string requestDevice
uint32 startFloor
float32 startX
float32 startY
uint32 endFloor
float32 endX
float32 endY
